src/stylesheets/components/_login.scss

```
@@ -0,0 +1,62 @@
+.login-form {
```

Use the UI element as a prefix (https://nimblehq.co/compass/development/code-conventions/css/#convention-25):

```suggestion
.form-login {
```

The file names must also match the CSS class name: `_form-login.scss`.